strr-web/pages/create-account-pre/[id].vue

```diff
@@ -119,6 +119,28 @@ const ownershipToApiType = (type: string | undefined): string => {
   return ''
 }
 
+const rentalUnitSpaceTypeToApiType = (type: string | undefined): RentalUnitSpaceTypeE | '' => {
+  if (type === 'ENTIRE_HOME') {
+    return RentalUnitSpaceTypeE.ENTIRE_HOME
+  } else if (type === 'SHARED_ACCOMMODATION') {
+    return RentalUnitSpaceTypeE.SHARED_ACCOMMODATION
+  }
+  return '' // Return empty string if no match
+}
+
+const hostResidenceToApiType = (hostResidence: string | undefined): HostResidenceE | '' => {
+  if (hostResidence === 'SAME_UNIT') {
```

Comment:
Can you just assume the any `hostResidence` string will be of type `HostResidenceE` and:
` return (hostResidence as HostResidenceE) || ''`